os/iphone.cmake

```diff
@@ -7,6 +7,8 @@ else()
   set(POLLY_OS_IPHONE_CMAKE 1)
 endif()
 
+include("${CMAKE_CURRENT_LIST_DIR}/../utilities/polly_status_debug.cmake")
```

Comment:
Can be just `include(polly_status_debug)`.

Reply:
We don't need this change.

Reply:
removed